### DISCUSSION TITLE
Yesterday everything worked, today there is an error.

### DISCUSSION THREAD

**Discussion by Kogavka** [illsk1lls/ZipRipper#35] [Q&A]
Yesterday everything worked, today there is an error. How to completely uninstall a program to reinstall it? How to clear all the files that it creates during operation?

Win11, ESET Internet Security, all updates
![Screenshot 2024-08-15 171847](https://github.com/user-attachments/assets/92e1f9af-7c61-4118-ba83-61ac88c0ee07)

**Comment by Kogavka** [comment]
I asked myself - I answered myself: in the Program Data folder, during initialization, a zipripper.png file of zero size is created, and after that an error pops up. If you download a file from the site or take it from the resources folder and put it instead of a file with a zero size, then the error disappears and the program works correctly. I have write rights, I'm running as administrator

**Comment by illsk1lls** [comment]
It might have been a temporary network issue, the download "should" work.. 

I would suggest creating zr-offline.txt by clicking JtR in Johns hat, that way you have an offline backup resource file, so you can run the script without downloading anything
